src/Compilers/CSharp/Portable/Symbols/PublicNullableAnnotationExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal static class PublicNullableAnnotationExtensions
+    {
+        public static NullableAnnotation ToInternalAnnotation(this CodeAnalysis.NullableAnnotation annotation)
```

Comment:
>ToInternalAnnotation [](start = 41, length = 20)

Where is this used? Aren't the public annotations a (read-only) view on the internal annotations?

Reply:
I think it was used at one point, but you're right, it's no longer used. I'll remove it.

---
In reply to: [265777820](https://github.com/dotnet/roslyn/pull/34017#discussion_r265777820) [](ancestors = 265777820)